com.chartboost.mediation/Runtime/AdFormats/Banner/Unity/ChartboostMediationUnityBannerAd.Creator.cs

```diff
@@ -0,0 +1,104 @@
+using Chartboost.Banner;
+using Chartboost.Utilities;
+using UnityEditor;
+using UnityEngine;
+using static Chartboost.Utilities.Constants;
+
+namespace Chartboost.AdFormats.Banner.Unity
+{
+    public partial class ChartboostMediationUnityBannerAd : MonoBehaviour
+    {
+#if UNITY_EDITOR
+        [MenuItem("Chartboost Mediation/UnityBannerAd/Create New")]
+        [MenuItem("GameObject/Chartboost Mediation/UnityBannerAd")]
```

Comment:
I can't imagine having two menu items for the same method works out very well in the Unity menu system.

Reply:
They both don't appear on the menu though. The one that starts from `GameObject` appears only in hierarchy ( on right-click)
<img src ="https://github.com/ChartBoost/chartboost-mediation-unity-sdk/assets/8779995/a1684feb-1a44-4010-815d-dd80d30123e0" width="500" height="400"/>